### PR TITLE
Update deployment pipeline for config

### PR DESCRIPTION
**Pull request by JackQuincy**
Updating deployment pipeline to use the yaml config.
Updating deployment pipeline to download the deployer instead of building it.
Updating deployment pipeline to download the config objects.
Removing orphaned shell script.

### CODE REVIEW

File: .pipelines/deploy-prod-env.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Azure DevOps Pipeline deploying the int environment
```
Comment (jim-minter) [inline .pipelines/deploy-prod-env.yml]: nit: the file is deploy-**prod**-env, the comment is for **int**.  Is it worth setting both to be ambiguous as it's for both int and prod?

Comment (JackQuincy) [reply .pipelines/deploy-prod-env.yml]: I can switch it to just be deploy-env.  I was trying to distinguish from dev. But I'm not sure the best word to do that

**Comment by JackQuincy** [comment]
Okay this is now failing inside the deployer instead of in the pipeline.  I think this is ready for review.

File: .pipelines/deploy-prod-env.yml (hunk 1 above)
Comment (jim-minter) [reply .pipelines/deploy-prod-env.yml]: deploy-env may not be the best, but I think it's good enough.

File: .pipelines/templates/template-deploy-azure-env.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      export AZURE_SUBSCRIPTION_ID="${{ parameters.subscriptionId }}"
+      export AZURE_CLIENT_ID=$(cat devops-spn.json | jq -r '.clientId')
+      export AZURE_CLIENT_SECRET=$(cat devops-spn.json | jq -r '.clientSecret')
+      export AZURE_TENANT_ID=$(cat devops-spn.json | jq -r '.tenantId')
+      rm devops-spn.json
+
+      export LOCATION=${{ parameters.location }}
+      export RESOURCEGROUP="not-in-use-moved-to-config"
+
+      export RP_VERSION=$(git rev-parse --short HEAD)
```
Comment (jim-minter) [inline .pipelines/templates/template-deploy-azure-env.yml]: should be removable

File: .pipelines/deploy-azure-env.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+          downloadType: 'specific'
+          downloadPath: '$(System.ArtifactsDirectory)/deployer'
+        displayName: "Download Deployer"
+      - template: ./templates/template-deploy-prod-env.yml
+        parameters:
+          workingDirectory: $(system.defaultWorkingDirectory)
+          configDirectory: '$(System.ArtifactsDirectory)/config/drop/deploy'
+          deployerDirectory: '$(System.ArtifactsDirectory)/deployer/drop'
+          configFileName: $(CONFIG_FILE_NAME)
+          location: $(LOCATION)
```
Comment (jim-minter) [inline .pipelines/deploy-azure-env.yml]: location parameter no longer used

File: .pipelines/templates/template-deploy-azure-env.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+      
+      echo ${{ parameters.azureDevOpsJSONSPN }} | base64 -d -w 0 > devops-spn.json
+      export AZURE_SUBSCRIPTION_ID="${{ parameters.subscriptionId }}"
+      export AZURE_CLIENT_ID=$(cat devops-spn.json | jq -r '.clientId')
+      export AZURE_CLIENT_SECRET=$(cat devops-spn.json | jq -r '.clientSecret')
+      export AZURE_TENANT_ID=$(cat devops-spn.json | jq -r '.tenantId')
+      rm devops-spn.json
+
+      chmod +x ${{ parameters.deployerDirectory }}/aro
+      ${{ parameters.deployerDirectory }}/aro deploy ${{ parameters.configDirectory }}/${{ parameters.configFileName }} ${{ parameters.location }}
```
Comment (JackQuincy) [inline .pipelines/templates/template-deploy-azure-env.yml]: used here

**Pull request merged**